generators/python/urllib3/templates/configuration.mustache

```diff
@@ -149,7 +150,11 @@ conf = {{{packageName}}}.Configuration(
 {{/hasHttpSignatureMethods}}
                  server_index=None,
                  server_operation_index=None, server_operation_variables=None,
+                 ignore_operation_servers=False,
                  ssl_ca_cert=None,
+                 retries=None,
```

Comment:
We should start using this?

Reply:
As stated on line 269, that should just be to override the default urllib behaviour (3 retries):
```python
        self.retries = retries
        """Adding retries to override urllib3 default value 3
        """
```
Anyway this configuration object is easily accessible by the customer, so that can also be easily customised. I believe we can go with the default value. Wdyt?